### DISCUSSION TITLE
会员服务应该提供一定的翻译服务免费额度吗？

### DISCUSSION THREAD

**Discussion by lmk123** [hcfyapp/crx-selection-translate#993] [讨论]
最近收到了一些关于会员功能的反馈，大致上就是都希望会员能有一定的免费翻译字符。这里我统一解释一下为什么我将会员功能和翻译服务分开了，也希望能收到一些反馈，看看大家觉得怎么做更好。

我确实考虑过把会员进行改造，给会员一定的免费字符数额度，但是：

这些翻译服务向我收费的时候是按照字符数收费的，而且，有道 / 搜狗翻译还没有每个月的免费额度；而划词翻译的会员本身很便宜， 3 元一个月，98元就永久了，如果给每个会员配置免费额度，这个价格是覆盖不了这些翻译服务的成本的。

举个例子，假设我每个月给会员 100 万字符额度，那么按照 40 元 / 百万字符计算，这 100 万字符额度的成本就已经有 40 元了，但我肯定不可能把会员设置成 40 元/ 月；如果设置成  50 万免费字符额度，那么成本为 20 元，但 20 元/月的会员也很难让人接受，而且 50 万字符完全不够用。

所以目前的形式就是将会员和翻译服务完全分开，翻译服务推荐自己申请密钥，或者用翻译积分计费，用多少买多少。

**Comment by zenof** [comment]
@lmk123  最近在使用 [OpenL](https://openl.club) 的服务，感觉还不错，也提供了 API，也许可以集成一下
